### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by jrfnl**
Includes:
* Installation instructions;
* Information on how to run and update the tooling;
* Information on how to use the WP-QA rulesets and how to interpret the results.

/cc @janw-me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,232 @@
...
+
+### Verify the install succeeded
+
+Once the installation is finished, run the following command to verify the installation was succesfull:
+```bash
+# For a project based install:
+vendor/bin/phpcs -i
+
+# For a global install:
+phpcs -i
```
Comment (janw-me) [inline README.md:98]: This may not work out of the box, the composer `vendor/bin` isn't necessarily in the default PATH.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,232 @@
...
+It is strongly recommended to document the settings you use in a custom ruleset and to add some minimal sniff configuration for optimal results.
+
+An example ruleset which you can place in the root of your project, including documentation on what you should adjust, can be found in the [sample-project-ruleset](https://github.com/jrfnl/QA-WP-Projects/tree/master/sample-project-ruleset) directory.
+
+
+## Reviewing WordPress plugins and themes
+
+This repository comes with two native rulesets - `WP-QA-Basic` and `WP-QA-Strict` - which are specifically intended for reviewing WordPress plugins and themes without much knowledge of code.
+
+These rulesets do not look at the code style consistency of code. They will only evaluate whether code is well documented, tested and whether there are any code quality issues detected.
```
Comment (janw-me) [inline README.md:134]: This peaks my interest, where _do_ I find the code style sniffs?
Maybe add a hint about those.

File: README.md (hunk 1 above)
Comment (jrfnl) [reply README.md:98]: I address that in the [Global Installation](https://github.com/jrfnl/QA-WP-Projects/blob/9790cc76dfd5301e938188b26ea579dce3097b22/README.md#global-installation) instructions.

> Make sure the Composer "home" `vendor/bin` directory is in your system path. To find out what the Composer "home" directory is, run `composer config --list --global` and see what's listed under `home`. Take that directory, add `/vendor/bin` to it and make sure it's in your operating system's `$PATH` variable.

If the command doesn't work, it basically means people didn't follow the installation instructions.

File: README.md (hunk 2 above)
Comment (jrfnl) [reply README.md]: In `WordPress-Core` and `WordPress-Extra` which are listed in the [Using the installed rulesets](https://github.com/jrfnl/QA-WP-Projects/blob/9790cc76dfd5301e938188b26ea579dce3097b22/README.md#using-the-installed-rulesets) section above.

I'll add a mention there that those include code style checks.

Comment (jrfnl) [reply README.md:134]: In `WordPress-Core` and `WordPress-Extra` which are listed in the [Using the installed rulesets](https://github.com/jrfnl/QA-WP-Projects/blob/9790cc76dfd5301e938188b26ea579dce3097b22/README.md#using-the-installed-rulesets) section above.

I'll add a mention there that those include code style checks.

**Comment by jrfnl** [comment]
@janw-me I've added a minor clarification. Do you think with my answers above and the clarification that this is good to go ?

**Comment by janw-me** [comment]
Yep seems fine by me :+1:

**Comment by jrfnl** [comment]
> 
> 
> Yep seems fine by me +1

Thanks for proof-reading :heart:

**Pull request merged**